ccd-definition/AuthorisationCaseEvent.json

```diff
@@ -153,6 +153,13 @@
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "CRU"
   },
+  {
```

Comment:
so if I am reviewing this to flag new functionality... all these R events for documents and courtadmin are new. They could not do this before your PR

Reply:
Will remove